src/poly/sampler.rs

```diff
@@ -50,10 +50,10 @@ pub trait PolyUniformSampler {
     ) -> Self::M;
 }
 
-pub trait PolyTrapdoorSampler: Send + Sync {
+pub trait PolyTrapdoorSampler {
```

Comment:
@rkdud007 
You removed `Send + Sync` because some new changes removes a reason to require the `PolyTrapdoorSampler ` to implement these traits, right?

Reply:
we just not needed Send + Sync on PolyTrapdoorSampler and noticed on this PR. not necessary from specific change